app/controllers/api/databases.php

```diff
@@ -2471,6 +2471,9 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
     ->action(function (string $range, Response $response, Database $dbForProject) {
 
         $usage = [];
+        if (App::getEnv('_APP_USAGE_STATS', 'enabled') != 'enabled') {
```

Comment:
I see code being repeated here. A better way to handle this is to create a new group called `usage` and create a new utopia `init` hook to check this logic. 


You can add these endpoints to a group using this
```diff
App::get('/v1/databases/usage')
    ->desc('Get usage stats for the database')
-    ->groups(['api', 'database'])
+    ->groups(['api', 'database', 'usage'])
    ->label('scope', 'collections.read')
    ->label('sdk.auth', [APP_AUTH_TYPE_ADMIN])
    ->label('sdk.namespace', 'databases')
    ->label('sdk.method', 'getUsage')
    ->label('sdk.response.code', Response::STATUS_CODE_OK)
    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
    ->label('sdk.response.model', Response::MODEL_USAGE_DATABASES)
    ->param('range', '30d', new WhiteList(['24h', '7d', '30d', '90d'], true), '`Date range.', true)
    ->inject('response')
    ->inject('dbForProject')
    ->action(function (string $range, Response $response, Database $dbForProject) {
       
    })
```


You can use these init hook as an example 
1. [Runs only for the `auth` group](https://github.com/appwrite/appwrite/blob/master/app/controllers/shared/api.php#L91)
2. [Runs only for the `project` group](https://github.com/appwrite/appwrite/blob/master/app/controllers/api/projects.php#L40)